CHANGELOG.rst

```diff
@@ -14,8 +14,9 @@ New features and notable changes:
 
 Bug fixes and small improvements:
 
-- Remove function coverage from sonarcube report (:issue:`591`)
-- Fix parallel processing of gcov data (:issue:`592`)
+- Remove function coverage from sonarcube report. (:issue:`591`)
+- Fix parallel processing of gcov data. (:issue:`592`)
+- Check resuired metadata keys. (:issue:`593`)
```

Comment:
Typo

```suggestion
- Check required metadata keys (:issue:`593`)
```

But it could make sense to reword the changelog line to be more helpful to end users. For example:

```suggestion
- Better diagnostics when dealing with corrupted input files (:issue:`593`)
```